src/config.ts

```diff
@@ -11,3 +11,13 @@ export const fallbackImages = {
 };
 
 export const REDIS_CHANGELOG_KEY = 'boot:latest_changelog';
+
+export enum StorageTopic {
+  Boot = 'boot',
+}
+
+export const generateStorageKey = (
+  topic: StorageTopic,
+  key: string,
+  identifier = 'global',
```

Comment:
Why do we set a default value here?

Reply:
Ah, in FE, it is common to have such values to be consistent across users. In BE, I realized the changelog was the same case, so I kept it. Though we can make it required here in BE since it is not as frequent as in FE.